exercises/scale-generator/canonical-data.json

```diff
@@ -0,0 +1,169 @@
+{
+  "exercise": "scale-generator",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "comments": [
+        "These tests have no interval.",
+        "Chromatic scale is considered the default scale"
+      ],
+      "description": "Chromatic scales",
+      "cases": [
+        {
+          "description": "Chromatic scale",
+          "property": "pitches",
+          "input": {
+            "tonic": "C"
+          },
+          "expected": [ "C", "C#", "D", "D#", "E", "F", "F#", "G", "G#", "A", "A#", "B" ]
+        },
+        {
+          "description": "Another chromatic scale",
```

Comment:
would it be helpful to note in the description that this chromatic scale contains flats instead of sharps? It would be more informative than "another"

Reply:
Good point. I'll find a better name.